tests/AssetManagerTest/View/Helper/AssetTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace AssetManagerTest\View\Helper;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use AssetManager\View\Helper\Asset;
+use Zend\ServiceManager\ServiceManager;
+
+class AssetTest extends TestCase
+{
+    public function testInvoke()
+    {
+        /* TODO need help
```

Comment:
@pensiero you need to test the helper, sorry. Feel free to poke me on hangouts about that, although I will be busy until Oct 25.